lib/discordrb/api/server.rb

```diff
@@ -200,17 +200,18 @@ def bans(token, server_id, limit = nil, before = nil, after = nil)
     )
   end
 
-  # Ban a user from a server and delete their messages from the last message_days days
+  # Ban a user from a server and delete their messages from seconds.
```

Comment:
This change does not make sense to me. Did you mean to write ".. the last message_seconds seconds."?

Reply:
I believe that originally "the last message_days days" would've been incorrect. I don't believe there's any need to list the parameter name in the description as it doesn't seem to be standard practice for this branch.

Reply:
I agree that the previous comment could have been worded better, but "delete their messages from seconds" sounds incorrect, since "seconds" isn’t some object that contains "their messages", such that they could be deleted from it. Rather, the user’s messages written after a point in time that was the specified amount of seconds ago are deleted.
How’ld something like `... and delete their messages written up to a given amount of time ago.` be?

Reply:
I have updated the wording to more closely align with what you've suggested.